apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -186,19 +186,32 @@ type ResourceFlavorReference string
 
 // ClusterQueueStatus defines the observed state of ClusterQueue
 type ClusterQueueStatus struct {
-	// flavorsUsage are the used quotas, by flavor, currently in use by the
+	// flavorsReservations are the reserved quotas, by flavor, currently in use by the
 	// workloads assigned to this ClusterQueue.
 	// +listType=map
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=16
 	// +optional
-	FlavorsUsage []FlavorUsage `json:"flavorsUsage"`
+	FlavorsReservations []FlavorUsage `json:"flavorsReservations"`
```

Comment:
```suggestion
	FlavorsReservation []FlavorUsage `json:"flavorsReservation"`
```

To use the consistent rule, this form would be better.